OmniBLE/Bluetooth/Pair/LTKExchanger.swift

```diff
@@ -22,7 +22,7 @@ class LTKExchanger {
     private let manager: PeripheralManager
     private let ids: Ids
```

Comment:
`RandomByteGenerator` is also a new protocol which OmniRandomByteGenerator conforms to. This allows for mocking fixed random values for testing.